lesson03/assignment/css/_components.content.scss

```diff
@@ -6,12 +6,55 @@ html {
 *:after,
 *:before {
     box-sizing: inherit;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.id__image {
+    border-radius: 50%;
+    max-width: 75%;
+}
+
+.card {
+  background-color: $white;
```

Comment:
Beyond the `max-width` comment from assignment02, I'd also suggest adding some top/bottom margin to the components to give them some visual separation on the page.